DrivewayDentDeletion/Operators/cicd-test/cicd-tasks.yaml

```diff
@@ -16,33 +16,9 @@ spec:
       description: The PVC to use for layers
   steps:
     - name: pull-retag-push-to-test
-      image: quay.io/buildah/stable:latest
-      securityContext:
-        privileged: true
+      image: quay.io/openshift/origin-cli:4.6.0
```

Comment:
Do we need to fix the cli version to 4.6.0 or can we use the latest?

Reply:
We're still using 4.6.0 everywhere (not just here). I don't think we can use latest, but probably could move this up a version. I think that's not for this PR though.